Dockerfile.tmpl

```diff
@@ -261,6 +261,10 @@ RUN pip install scipy \
     apt-get install -y pandoc && \
     pip install essentia
 
+# #1281 Install numba MVC support:
```

Comment:
@djherbis Do you have a bug for adding numba MVC support with where you found the instructions?

Tests are failing with:

```
  File "/opt/conda/lib/python3.10/site-packages/numba/cuda/codegen.py", line 159, in get_cubin
    linker.add_ptx(ptx.encode())
  File "/opt/conda/lib/python3.10/site-packages/numba/cuda/cudadrv/driver.py", line 2710, in add_ptx
    compile_result = compile_ptx(ptx.decode(), self.ptx_compile_options)
  File "/opt/conda/lib/python3.10/site-packages/ptxcompiler/api.py", line 33, in compile_ptx
    raise RuntimeError(error_log)
RuntimeError: ptxas application ptx input, line 9; fatal   : Unsupported .version 8.0; current version is '7.8'
```

Which seems to indicate an incompatible version of ptx.

Also, this is installing cuda (gpu) specific libraries and should be inside a `{{ if eq .Accelerator "gpu" }} ... {{ end }}` block.

Reply:
I found this guide which is probably where you got the instructions from: https://numba.readthedocs.io/en/stable/cuda/minor_version_compatibility.html#enabling-mvc-support